src/features/collaborator-avatars/CollaboratorListItem.scss

```diff
@@ -74,3 +74,7 @@
         }
     }
 }
+
+.bdl-CollaboratorListItem-user {
```

Comment:
Should we move this within `collaborator-list-item`? Does it make sense to introduce the standard `bdl-CollaboratorListItem` when all the other classes don't follow this pattern in this file?

Reply:
this file has super weird classes `info`, `role`, `user`, etc.

normally I think it might be okay to use the current file's pattern but because these are so generic that it'll likely conflict with customer's classes that I think it'd be good to lean towards using new classnames in this case

Reply:
> Should we move this within `collaborator-list-item`?

with the new SUIT class, I left it outside of the parent scope block so that it doesn't have the extra specificity

Reply:
Ok sounds good